runtest.py

```diff
@@ -190,14 +190,19 @@ def run_test_result_analyzer(self, test_case_module_name, test_name, current_run
 
                         current_run += 1
                 else:
-                    if run_result.get("comparing_image_missing", False):
-                        if "time_list_counter" in run_result:
-                            current_run = int(run_result['time_list_counter'])
+                    if objStatusRecorder.STATUS_IMG_COMPARE_RESULT in status_result:
+                        if status_result[objStatusRecorder.STATUS_IMG_COMPARE_RESULT] == objStatusRecorder.PASS_IMG_COMPARE_RESULT:
```

Comment:
I'd prefer to add the comments here for explain why we split these two `if` condition here.